src/components/ui/Icon.vue

```diff
@@ -455,12 +455,21 @@
 </template>
 
 <script>
+const viewBoxValues = {
+  staff: '5 5 14 14',
```

Comment:
![image](https://user-images.githubusercontent.com/3604775/63431007-2f1cf280-c41e-11e9-9ff4-081a7efbab82.png)
This breaks the original use cases for the staff icon.

Reply:
I've made an adjustment here for that. Good catch!